.github/workflows/ci.yml

```diff
@@ -301,122 +243,132 @@ jobs:
-                  docker/sdk testing codecept run -c codeception.acceptance.yml
-
-
-    php-81-mariadb-functional-alpine:
-        name: "PHP 8.1 / MariaDB / Functional / Alpine"
-        runs-on: ubuntu-20.04
-        env:
-            PROGRESS_TYPE: plain
-            SPRYKER_PLATFORM_IMAGE: spryker/php:8.1
-            TRAVIS: 1
-
-        steps:
-            - uses: actions/checkout@v3
+            - uses: actions/setup-python@v4
+              with:
+                  python-version: '3.9'
 
-            - name: Install apt-packages
+            - name: Install packages
               run: |
+                  sudo apt-get update
                   sudo apt-get install apache2-utils
-
+                  sudo apt install awscli -q
+                  python3 -m pip install --upgrade pip
+                  python3 -m pip install -U robotframework
+                  python3 -m pip install -U robotframework-requests
+                  python3 -m pip install -U robotframework-jsonlibrary
+                  python3 -m pip install -U robotframework-databaselibrary
+                  python3 -m pip install PyMySQL
             - name: Install docker-compose
               run: |
                   sudo curl -L "https://github.com/docker/compose/releases/download/2.12.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
                   sudo chmod +x /usr/local/bin/docker-compose
-
-            - name: Run docker
+            - name: Install Robot tests
+              run: |
+                  git clone https://github.com/spryker/robotframework-suite-tests.git --single-branch --branch master robotframework-tests
+            - name: Install Project
+              continue-on-error: true
               run: |
                   git clone https://github.com/spryker/docker-sdk.git ./docker
-                  docker/sdk boot deploy.ci.functional.mariadb.yml -v
+                  docker/sdk boot -v deploy.ci.acceptance.mariadb.dynamic-store.yml
+                  sudo bash -c "echo '127.0.0.1 backend-api.eu.spryker.local backend-api.us.spryker.local backend-gateway.eu.spryker.local backend-gateway.us.spryker.local backoffice.eu.spryker.local backoffice.us.spryker.local glue-backend.eu.spryker.local glue-backend.us.spryker.local glue-storefront.eu.spryker.local glue-storefront.us.spryker.local glue.eu.spryker.local glue.us.spryker.local mail.spryker.local mp.eu.spryker.local mp.us.spryker.local queue.spryker.local scheduler.spryker.local spryker.local swagger.spryker.local yves.eu.spryker.local yves.us.spryker.local' >> /etc/hosts"
                   docker/sdk up -t -v
-                  docker/sdk testing codecept run -c codeception.functional.yml
-
+            -   name: Custom commands
+                continue-on-error: false
+                run: |
+                    docker/sdk cli composer dump-autoload -o -a --apcu
+                    SPRYKER_CURRENT_REGION=EU docker/sdk console queue:worker:start --stop-when-empty
```

Comment:
Don't we have to add `SPRYKER_DYNAMIC_STORE_MODE=true` as well like it was done [here](https://github.com/spryker/suite-nonsplit/pull/8831/files#diff-bf1dacc49ef3c3cff2ab96574796c0f9e5284b2743f2745a260b062bccb45ec0R61) for DMS jobs?

Reply:
There it is used for the different commands.
In the current case, we use specificc deployment file for dynamic store.
Ofc we can add it, but it will change nothing.